skimage/transform/_warps.py

```diff
@@ -18,14 +17,27 @@
 )
 
 
+def _multichannel_default(multichannel, ndim):
+    if multichannel is not None:
+        return multichannel
+    else:
+        warn('The default multichannel argument (None) is deprecated. '
+             'Specifiy True or False explicitly.')
+        # utility for maintaining previous color image default behavior
+        if ndim == 3:
+            return True
+        else:
+            return False
+
+
 def resize(image, output_shape, order=1, mode=None, cval=0, clip=True,
            preserve_range=False):
```

Comment:
> This means that the multichannel nature can be inferred from the output and input shapes.
A-a-a-and we are back to it :).

@jni I understand your point. My feeling is that the API should be as homogeneous as possible.
Maybe we could just have another input from @scikit-image/core ?